js/apps/account-ui/test/applications.spec.ts

```diff
@@ -62,15 +64,15 @@ test.describe("Applications test", () => {
       "Skip this test if not running with regular Keycloak",
     );
 
-    await login(page, "admin", "admin", "master");
+    await login(page, "admin", "admin");
 
     // go to admin console
     await page.goto("/");
-    await expect(page).toHaveURL("http://localhost:8080/admin/master/console/");
-    await page.waitForURL("http://localhost:8080/admin/master/console/");
+    await expect(page).toHaveURL(`${getBaseUrl()}/admin/master/console/`);
+    await page.waitForURL(`${getBaseUrl()}/admin/master/console/`);
 
-    await page.goto("/realms/master/account");
-    await page.waitForURL("http://localhost:8080/realms/master/account/");
+    await page.goto(getRootPath());
+    await page.waitForURL(getBaseUrl() + getRootPath());
```

Comment:
Nit: if instead of `getRootPath()` there was a function `getAccountPath()` in `utils.ts`, and there was a similar `getAdminPath()` function, this could be rewritten to:

```suggestion
    await expect(page).toHaveURL(getBaseUrl() + getAdminPath());
    await page.waitForURL(getBaseUrl() + getAdminPath());

    await page.goto(getRootPath());
    await page.waitForURL(getBaseUrl() + getAccountPath());
```

or even simpler with additional functions `getAccountUrl`/`getAdminUrl` which would concatenate base url with the respective path.

```suggestion
    await expect(page).toHaveURL(getAdminUrl());
    await page.waitForURL(getAdminUrl());

    await page.goto(getRootPath());
    await page.waitForURL(getAccountUrl());
```

Reply:
Yeah, this could be pulled into the `utils` a little more, I've pushed some changes, let me know what you think.